kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java

```diff
@@ -3,40 +3,41 @@
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.SelenideElement;
 import io.qameta.allure.Step;
-import org.openqa.selenium.By;
 
-import static com.codeborne.selenide.Selenide.$;
 import static com.codeborne.selenide.Selenide.$x;
 import static com.provectus.kafka.ui.utilities.WebUtils.clickByJavaScript;
 import static com.provectus.kafka.ui.utilities.WebUtils.isVisible;
 
 public class SchemaRegistryList {
 
-    private final SelenideElement schemaButton = $(By.xpath("//*[contains(text(),'Create Schema')]"));
+    protected SelenideElement loadingSpinner = $x("//*[contains(text(),'Loading')]");
+    protected SelenideElement createSchemaBtn = $x("//button[contains(text(),'Create Schema')]");
+    protected SelenideElement schemaGrid = $x("//table");
+    protected String schemaElementLocator = "//a[contains(text(),'%s')]";
 
     @Step
     public SchemaRegistryList waitUntilScreenReady(){
-        $x("//*[contains(text(),'Loading')]").shouldBe(Condition.disappear);
-        $x("//button[contains(text(),'Create Schema')]").shouldBe(Condition.visible);
+        loadingSpinner.shouldBe(Condition.disappear);
+        createSchemaBtn.shouldBe(Condition.visible);
         return this;
     }
 
     @Step
     public SchemaCreateForm clickCreateSchema() {
```

Comment:
pls change the method to return SchemaCreateForm

Reply:
Refactored to return SchemaRegistryList